pkgs/development/haskell-modules/configuration-ghc-9.12.x.nix

```diff
@@ -57,6 +57,25 @@ self: super: {
   unix = null;
   xhtml = null;
 
+  doctest = lib.pipe super.doctest [
+    (overrideSrc {
+      # Not yet on hackage: https://github.com/sol/doctest/pull/462
+      src = pkgs.fetchFromGitHub {
+        owner = "sol";
+        repo = "doctest";
+        rev = "77373c5d84cd5e59ea86ec30b9ada874f50fad9e";
+        hash = "sha256-nPfVDnHlKL2DYo9+IDKxA2AvRnjLQ8kr8YGHHD9APvM=";
+      };
+    })
+    (overrideCabal (drv: {
+      testFlags = drv.testFlags or [] ++ [
+        # These tests require cabal-install
+        "--skip=/Cabal.ReplOptions"
```

Comment:
These tests were added to a newer version than LTS so I defaulted to a 9.12 only override, but doctest seemed to build even on 9.8 if I instead added  the `skip` to https://github.com/NixOS/nixpkgs/blob/43c0e522351535cefbc02636c1e17dd9f752681c/pkgs/development/haskell-modules/configuration-nix.nix#L1497-L1504
Maybe we should add it there for simplicity given this would eventually propagate anyway in 2 major LTS releases ? It causes ghc 9.12 itself to be rebuilt, seemingly due to 
`shake -> quickcheck-instances -> extra -> vector -> doctest`